Pylint: R1716

```bash
mmpose/datasets/pipelines/bottom_up_transform.py:168:20: R1716: Simplify chained comparison between the operands (chained-comparison)
mmpose/models/backbones/regnet.py:135:15: R1716: Simplify chained comparison between the operands (chained-comparison)
mmpose/models/backbones/resnet.py:503:15: R1716: Simplify chained comparison between the operands (chained-comparison)
```

--- a/mmpose/datasets/pipelines/bottom_up_transform.py
+++ b/mmpose/datasets/pipelines/bottom_up_transform.py
@@ -166,7 +166,7 @@ class JointsEncoder():
             for idx, pt in enumerate(joints[i]):
                 x, y = int(pt[0]), int(pt[1])
                 if (pt[2] > 0 and 0 <= y < self.output_res
-                        and  0 <= x < self.output_res):
+                        and 0 <= x < self.output_res):
                     if self.tag_per_joint:
                         visible_kpts[i][tot] = \
                             (idx * output_res**2 + y * output_res + x, 1)

--- a/mmpose/datasets/pipelines/bottom_up_transform.py
+++ b/mmpose/datasets/pipelines/bottom_up_transform.py
@@ -165,8 +165,8 @@ class JointsEncoder():
             tot = 0
             for idx, pt in enumerate(joints[i]):
                 x, y = int(pt[0]), int(pt[1])
-                if (pt[2] > 0 and x >= 0 and y >= 0 and x < self.output_res
-                        and y < self.output_res):
+                if (pt[2] > 0 and self.output_res > y >= 0
+                        and self.output_res > x >= 0):
                     if self.tag_per_joint:
                         visible_kpts[i][tot] = \
                             (idx * output_res**2 + y * output_res + x, 1)

--- a/mmpose/datasets/pipelines/bottom_up_transform.py
+++ b/mmpose/datasets/pipelines/bottom_up_transform.py
@@ -165,8 +165,8 @@ class JointsEncoder():
             tot = 0
             for idx, pt in enumerate(joints[i]):
                 x, y = int(pt[0]), int(pt[1])
-                if (pt[2] > 0 and self.output_res > y >= 0
-                        and self.output_res > x >= 0):
+                if (pt[2] > 0 and 0 <= y < self.output_res
+                        and  0 <= x < self.output_res):
                     if self.tag_per_joint:
                         visible_kpts[i][tot] = \
                             (idx * output_res**2 + y * output_res + x, 1)

--- a/mmpose/models/backbones/regnet.py
+++ b/mmpose/models/backbones/regnet.py
@@ -132,7 +132,7 @@ class RegNet(ResNet):
         self.stem_channels = stem_channels
         self.base_channels = base_channels
         self.num_stages = num_stages
-        assert num_stages >= 1 and num_stages <= 4
+        assert 4 >= num_stages >= 1
         self.strides = strides
         self.dilations = dilations
         assert len(strides) == len(dilations) == num_stages

--- a/mmpose/models/backbones/regnet.py
+++ b/mmpose/models/backbones/regnet.py
@@ -132,7 +132,7 @@ class RegNet(ResNet):
         self.stem_channels = stem_channels
         self.base_channels = base_channels
         self.num_stages = num_stages
-        assert 4 >= num_stages >= 1
+        assert 1 <= num_stages <= 4
         self.strides = strides
         self.dilations = dilations
         assert len(strides) == len(dilations) == num_stages

--- a/mmpose/models/backbones/resnet.py
+++ b/mmpose/models/backbones/resnet.py
@@ -500,7 +500,7 @@ class ResNet(BaseBackbone):
         self.stem_channels = stem_channels
         self.base_channels = base_channels
         self.num_stages = num_stages
-        assert num_stages >= 1 and num_stages <= 4
+        assert 4 >= num_stages >= 1
         self.strides = strides
         self.dilations = dilations
         assert len(strides) == len(dilations) == num_stages

--- a/mmpose/models/backbones/resnet.py
+++ b/mmpose/models/backbones/resnet.py
@@ -500,7 +500,7 @@ class ResNet(BaseBackbone):
         self.stem_channels = stem_channels
         self.base_channels = base_channels
         self.num_stages = num_stages
-        assert 4 >= num_stages >= 1
+        assert 1 <= num_stages <= 4
         self.strides = strides
         self.dilations = dilations
         assert len(strides) == len(dilations) == num_stages

